apps/qubit/modules/settings/templates/pageElementsSuccess.php

```diff
@@ -54,7 +54,7 @@
               <?php echo $form->toggleDigitalObjectMap->label('Digital object map')->renderLabel() ?>
               <?php if (!$googleMapsApiKeySet): ?>
                 <div class="description">
-                  <?php $securitySettingsLinkHtml = link_to(__('Security'), array('module' => 'settings', 'action' => 'security')) ?>
+                  <?php $securitySettingsLinkHtml = link_to(__('global'), array('module' => 'settings', 'action' => 'global')) ?>
                   <?php echo __('This feature will not work until a Google Maps API key is specified on the %1% settings page.', array('%1%' => $securitySettingsLinkHtml)) ?>
```

Comment:
There is a way to merge these two i18n strings as one. You have one example here: https://github.com/artefactual/atom/blob/1d80f3826fcbdf318a8c99fa37fc2d33da42fbcd/plugins/sfIsdiahPlugin/modules/sfIsdiahPlugin/actions/indexAction.class.php#L71